### PR TITLE
Resolve previous PR #50

### PR DESCRIPTION
**Pull request by aminalhazwani**
The previous PR was missing the compiled files and the version bump.

### CODE REVIEW

File: photon-colors/photon-colors.swift [hunk 1]
```diff
@@ -2,86 +2,91 @@
...
-        static let Magenta50 = UIColor(rgb: 0xff1ad9)
-        static let Magenta60 = UIColor(rgb: 0xed00b5)
-        static let Magenta70 = UIColor(rgb: 0xb5007f)
-        static let Magenta80 = UIColor(rgb: 0x7d004f)
-        static let Magenta90 = UIColor(rgb: 0x440027)
+        static let Magenta50 = UIColor(rgb: 0xff1ad9);
+        static let Magenta60 = UIColor(rgb: 0xed00b5);
+        static let Magenta70 = UIColor(rgb: 0xb5007f);
+        static let Magenta80 = UIColor(rgb: 0x7d004f);
+        static let Magenta90 = UIColor(rgb: 0x440027);
```
Comment (bwinton) [inline photon-colors/photon-colors.swift]: There shouldn't be semicolons here… Did you run an older version of the script?

**Review by bwinton — CHANGES_REQUESTED** [review]

File: photon-colors/photon-colors.swift (hunk 1 above)
Comment (aminalhazwani) [reply photon-colors/photon-colors.swift]: Fix with bccff2e ☺️

**Review by bwinton — APPROVED** [review]

**Comment by bwinton** [comment]
Okay two more things.
1) A changelog detailing what you did, and
2) a tag for the version so people can compare it with the previous.
😉

**Comment by aminalhazwani** [comment]
@bwinton do you prefer a new file like  `CHANGELOG.md` in the repository or shall we use the GitHub built in feature? Thanks!

**Comment by bwinton** [comment]
Either works for me, so I would say go with whatever's easiest. :)

**Comment by aminalhazwani** [comment]
I suppose I was too late to  the game, I prepared a first README but did not pushed it. I will follow up with a new PR, thanks @bwinton!

**Pull request merged**